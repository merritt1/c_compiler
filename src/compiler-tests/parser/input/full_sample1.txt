char fcn(unsigned *a[10+2]);
int a[];


int main(void) {
    a = b+c/d++*e;
    j = a[5];
    print("Hello");
    potato: ++a;
    return a;
}
