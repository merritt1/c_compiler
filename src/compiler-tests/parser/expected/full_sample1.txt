signed char (fcn(unsigned int (*(a[(10+2)]))));
signed int (a[]);
signed int (main(void)) {
(a=(b+((c/(d++))*e)));
(j=(*(a+5)));
(print("Hello"));
potato:
    (++a);
return a;
}
